odoo/tests/common.py

```diff
@@ -224,6 +225,62 @@ def assertQueryCount(self, default=0, margin=0, **counters):
         else:
             yield
 
+    def assertRecordsEqual(self, records, expected_values):
+        ''' Compare a recordset with a list of dictionaries representing the expected results.
+
+        Note that:
+          - Comparison between falsy values is supported: False match with None.
+          - Comparison between monetary field is also treated according the currency's rounding.
+          - Comparison between x2many field is done by ids. Then, empty expected ids must be [].
+          - Comparison between many2one field id done by id. Empty comparison can be done using any falsy value.
+
+        :param records:               The records to compare.
+        :param expected_values:       List of dicts expected to be exactly matched in records
+        '''
```

Comment:
looks okay to me. can you just add in the docstring that the comparison is done element by element, index-wise (if that makes any sense)?

Also, please add an extra kwarg `sorting_function`, if given it will call sorted(key=sorting_function) on the recordset prior to the current code

Reply:
I'm not sure it's really relevant since you could simply sort your records before calling the function. If you adapt the accounting tests, you don't need it for example.

Reply:
Docstring updated